ocp_resources/user_defined_network.py

```diff
@@ -128,3 +139,88 @@ def to_dict(self) -> None:
 
             if self.ipam_lifecycle:
                 _layer2["ipamLifecycle"] = self.ipam_lifecycle
+
+
+class Layer3Subnets:
```

Comment:
I don't see any reason to have this class, you can pass a list of subnets ([{},{}]) to `Layer3UserDefinedNetwork` directly

Reply:
Yes we can, I added mimic the API structure as close as possible.
If you want I will remove this class.